### DISCUSSION TITLE
Agregar Cadastro de peças, produtos,equipamentos e imobilizado de Empresas

### DISCUSSION THREAD

**Discussion by sergiocandidoc** [RamonSilva20/mapos#2518] [Ideas]
Gostaria de sugerir que um sistema desses já até mesmo abandonados no GIT Hub fossem agregados incorporados ao MAPOS assim o sistema de ordem de serviços tendo inventário se tornarria extremamente útil para empresas que podem controlar a manutenção cadastro, depreciação e outros de seu parque de equipamentos. Já testei alçguns e faltam funções que o MAPOS tem e podem ser melhorados e ainda usando o mesmo DB se forem adaptados ao MAPOS o que acham?

**Comment by marlontomaz** [comment]
boa tarde, achei uma boa ideia... mas vc pensou em deixar na mesma tabela de cadastro de produtos e separar por categoria exemplo: venda , mobilizado ? ou criar uma tabela de cadastro de inventario ?
